docs/NewTargetSpecificNode.md

```diff
@@ -0,0 +1,93 @@
+## New Target Specific Node
+
+
+This document describes how to add new target specific node to make further optimization on certain backend.
+
+### Graph Optimization
+
+Glow has two levels of IR. The high-level IR is a dataflow node-based graph. There are several strategies to make transformation on high-level IR to achieve better performance on certain backend. Both these strategies will introduce new target specific node.
```

Comment:
Instead of "Both these strategies", I would say "Two of these strategies", just to make clear that there are other ways to optimize the graph that do not require backend-specific nodes.

Reply:
Done